jscomp/gentype_tests/typescript-react-example/src/FirstClassModulesInterface.gen.tsx

```diff
@@ -3,13 +3,12 @@
 /* eslint-disable */
 /* tslint:disable */
 
-import * as FirstClassModulesInterfaceBS__Es6Import from './FirstClassModulesInterface.bs';
-const FirstClassModulesInterfaceBS: any = FirstClassModulesInterfaceBS__Es6Import;
+import * as FirstClassModulesInterfaceBS from './FirstClassModulesInterface.bs';
 
 export type record = { readonly x: number; readonly y: string };
 
 export type firstClassModule = { readonly x: number };
 
-export const MT_x: number = FirstClassModulesInterfaceBS.MT.x;
```

Comment:
TypeScript report on this

```
error TS2339: Property 'MT' does not exist on type 'typeof import("/Users/runner/work/rescript-compiler/rescript-compiler/jscomp/gentype_tests/typescript-react-example/src/FirstClassModulesInterface.bs")'
```

because `MT` is actually module type, so it should be handled as a type, not a value